README.md

```diff
@@ -9,3 +9,13 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+Two conditions that may be checked before testing directly for isomorphism are testing if the two graphs have the same number of vertices/nodes and if the list of the degrees of each node of the two graphs are equal to each other. Our two conditions for graphs $A$ and $B$ are that they do have the same number of nodes and that they are completely connected. By definition, a completely connected graph (or complete graph) states that there is an "edge between every pair of vertices." This means that each node/vertex will have $v-1$ edges (where $v$ is the number of vertices in the graph) because it will point to each vertex except itself. Thus, both graphs $A$ and $B$ have degrees of $v-1$ for all of their nodes. Therefore the two predonditions to testing isomorphism have been satisfied. 
+
+The main test for isomorpism is if there exists a bijective function that would transform the vertices of one graph to another such that the two graphs would become equivalent in terms of vertices and edges. This assumes the "edge structure" is the same and that only the vertices need rearranged to make the graphs equal. Going back to the graphs being completely connected, if from one vertex there is an edge to every other vertex for all vertices, then that means the two graphs will have the same "edge structure." Each node has a path to every other node. Then there exists a function that rearranges the vertices to match the vertices and edges of the other graphs (as the vertices only need to be rearranged). The mapping must exist because every single vertex connects to every other one for each vertex, giving all the vertices the same properties as all of the others. The function will be one-to-one and onto because each vertex in the first graph will map to exactly on vertex in the second graph. 
```

Comment:
I can easily construct a function that is not one-to-one and onto -- how do you know that you can construct the right function?